### ISSUE TITLE
ZBOSS Zigbee network backup and network restore via zigpy radio API?

### ISSUE THREAD

**Issue by Hedda** [kardia-as/zigpy-zboss#11]
zigpy's (new) radio API support unified commands for Zigbee network backup and network restore; does/will [zigpy-zboss](https://github.com/kardia-as/zigpy-zboss) support this?

* https://github.com/zigpy/zigpy-cli/blob/dev/README.md#network-commands
  * https://github.com/zigpy/zigpy-cli/blob/dev/README.md#network-backup
    * https://github.com/zigpy/zigpy-cli/blob/dev/README.md#network-restore

zigpy uses the self-developed open standard and open file format specified in the "Open ZigBee Coordinator Backup Format" project:

https://github.com/zigpy/open-coordinator-backup

Zigbee network backup to file and restore is also an important feature because it allows network migration to and from different radios.

* https://skyconnect.home-assistant.io/zigbee-migration-selection/
  * https://skyconnect.home-assistant.io/migrate-zigbee-zha-only/

Reference:

https://github.com/zigpy/zigpy-cli/pull/2
https://github.com/zigpy/zigpy/pull/1006
https://github.com/zigpy/zigpy-cli/pull/13
https://github.com/zigpy/zigpy-znp/pull/97
https://github.com/zigpy/bellows/pull/445
https://github.com/zigpy/zigpy-deconz/pull/172
https://github.com/zigpy/zigpy-zigate/pull/117

PS: Some zigpy radio libraries, like zigpy-znp and bellows, also have extended CLI tools to perform full NVRAM backup and restore too, as well as other functions, such as for example NVRAM reset (which can also be useful before doing a restore from backup):

* https://github.com/zigpy/zigpy-znp/blob/dev/TOOLS.md#nvram-backup

* https://github.com/zigpy/bellows/blob/dev/README.md#nvram-backup-and-restore

PPS: In addition suggest check out the unofficial “zha-toolkit” which adds additional services to ZHA (including for backup):

https://github.com/mdeweerd/zha-toolkit

https://community.home-assistant.io/t/zha-toolkit-a-big-set-of-zigbee-commands-on-top-of-zha-zigpy/373346

**Comment by DamKast** [comment]
Yes, it will fix it I think. I didn't test thoroughly but I tested network migration from a nRF52840DK to a nRF52840_dongle and it worked properly.

@genesiscrew and I are working on unit tests, I thought of releasing a new version after this.

**Comment by Hedda** [comment]
Should also test that possible to migrate between different radio libaries, so go from zigpy-zboss to bellows/zigpy-znp and vice verse, for back and forth migration.
